src/Features/Core/Portable/ConvertToInterpolatedString/AbstractConvertPlaceholderToInterpolatedStringRefactoringProvider.cs

```diff
@@ -57,28 +61,83 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 return;
             }
 
-            var (invocation, invocationSymbol) = await TryFindInvocationAsync(textSpan, document, semanticModel, formatMethods, syntaxFactsService, context.CancellationToken).ConfigureAwait(false);
-            if (invocation != null && invocationSymbol != null &&
-                IsArgumentListCorrect(syntaxFactsService.GetArgumentsOfInvocationExpression(invocation), invocationSymbol, formatMethods, semanticModel, syntaxFactsService, cancellationToken))
+            var addedRefactorings = await TryAddRefactoringsAsync(
```

Comment:
I don't know how slow this is, to know if its a problem to call it twice, but an alternate suggestion:

```cs
var allMethods = replaceInvocationMethods.AddRange(keepInvocationMethds);
var (invocation, invocationSymbol) = await TryFindInvocationAsync(..., allMethods ...);
var shouldReplaceInvocation = (keepInvocationMethods.Contains(invocation);
...
```

If that makes any sense? I thought of it while watching the stream but couldn't type it out in twitch chat.

Reply:
hmm, that's a good question. It's really a problem of what is slower. Let me look further into this. Agree that the implementation is sus right now